docpages/advanced_reference/coroutines.md

```diff
@@ -1,50 +1,38 @@
 \page coroutines Advanced commands with coroutines
 
-\warning D++ Coroutines are a very new feature and are currently only supported by D++ on g++ 13.1 and MSVC 19.37, and the CMake option DPP_CORO must be enabled. They are experimental and may have bugs or even crashes, pease report any to [GitHub Issues](https://github.com/brainboxdotcc/DPP/issues) or to our [Discord Server](https://discord.gg/dpp).
+\warning D++ Coroutines are a very new feature and are currently only supported by D++ on g++ 13.1 and MSVC 19.37, and the CMake option DPP_CORO must be enabled. They are experimental and may have bugs or even crashes, please report any to [GitHub Issues](https://github.com/brainboxdotcc/DPP/issues) or to our [Discord Server](https://discord.gg/dpp).
 
 ### What is a coroutine?
 
 Introduced in C++20, coroutines are the solution to the impracticality of callbacks. In short, a coroutine is a function that can be paused and resumed later : they are an extremely powerful alternative to callbacks for asynchronous APIs in particular, as the function can be paused when waiting for an API response, and resumed when it is received.
-
 Let's revisit [attaching a downloaded file](/attach-file.html), but this time with a coroutine :
-
-
 ~~~~~~~~~~~~~~~{.cpp}
 #include <dpp/dpp.h>
-

```

Comment:
Personally imo the blank lines in codes doesn't need to be removed since it improves code readability

Reply:
idk maybe it is original. I just redo it

Reply:
yes blank lines do not need to be removed

Reply:
And especially in the text outside of the code. Blank line starts a new paragraph in Markdown.